Library/Formula/hspell.rb

```diff
@@ -0,0 +1,85 @@
+class Hspell < Formula
+  desc "The Hspell project is a free Hebrew linguistic project."
+  homepage "http://hspell.ivrix.org.il/"
+  url "http://hspell.ivrix.org.il/hspell-1.3.tar.gz"
+  sha256 "603c25dcbaa65d171d9065da7369cfe0dc21bda8378bade13b42eda69c8b2fe7"
+
+  depends_on "autoconf" => :build
+
+  # hspell was built for linux and compiles a .so shared library, to comply with macOS
+  # standards this patch creates a .dylib instead
+  patch :p0, :DATA
+
+  def install
+    ENV.deparallelize
+
+    # autoconf needs to pick up on the patched configure.in and create a new ./configure
+    # script
+    system "/usr/local/bin/autoconf"
```

Comment:
Please use `system "autoconf"` instead of the full path. Homebrew can be installed outside of `/usr/local`.